docs/benchmarks.md

```diff
@@ -0,0 +1,57 @@
+# GCSFuse Performance Benchmarks
+
+FIO is used to perform load tests on GCSFuse. Below tables shows performance metrics of GCSFuse for different workloads for the given test setup:
```

Comment:
Can link to https://fio.readthedocs.io/en/latest/ ?

Reply:
done